apps/dsp-app/src/app/workspace/resource/resource.component.html

```diff
@@ -23,6 +23,7 @@
         [currentTab]="selectedTabLabel"
         [parentResource]="incomingResource ? incomingResource.res : resource.res"
         [activateRegion]="selectedRegion"
+        [adminPermissions]="isAdmin$ | async"
```

Comment:
You don't need to add an input adminPermission if you use the admin selector. You can add it straight to the component ?

Reply:
There is no equivalent selector yet as far as I can see (it is not sysadmin, it is sysadmin or project admin/member of the current project). So I went for a short streamlined fix reusing what is already there instead of refactoring and implementing new stuff as well.